### ISSUE TITLE
The roadmap includes courses behind a paywall

### ISSUE THREAD

**Issue by bigfatbird** [MoienTajik/AspNetCore-Developer-Roadmap#113]
While this roadmap might be great I don't think it should include courses behind a paywall/subscription service.

For example the first course uses pluralsight, which is not a free resource. An alternative should be found.

**Comment by kkokosa** [comment]
But this is community-driven project. If "should be found", find it and contribute.

**Comment by bigfatbird** [comment]
@kkokosa Of course, I understand that. But there should be a policy in place to not include resources that have such a high burden to enter

**Comment by kkokosa** [comment]
Well, not agree. Maybe someone is willing to spend extra money to get extra content? What can be done is to just some additional info/icon for paid content 🤔

**Comment by bigfatbird** [comment]
@kkokosa I think that is a step in the right direction, though it is not very inclusive. Some people just really don't have money, and education should be free, or as cheap as possible in my opinion, but that discussion is off topic.

I see the value in paid courses, for sure.

**Comment by Holiv** [comment]
@kkokosa it was my case. I found out about pluralsight through this roadmap, i decided to pay for it and i have no regrets until the moment.
